src/httpClient.js

```diff
@@ -43,7 +43,10 @@ requestFetch.interceptors.response.use((response) => {
     }
     else {
         store.dispatch('loader/hide')
-        store.dispatch('error/displayErrorMessage', 'An error occurred. Please try again later.');
+        store.dispatch('error/displayErrorMessage', 'Your session has expired!');
```

Comment:
please show this message when session expired

Reply:
done. but response.status for this feature is 0 not 401